samplers/aws/xray/remote_sampler_config.go

```diff
@@ -0,0 +1,109 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package xray // import "go.opentelemetry.io/contrib/samplers/aws/xray"
+
+import (
+	"fmt"
+	"log"
+	"math"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/go-logr/stdr"
+)
+
+const (
+	defaultProxyEndpoint   = "127.0.0.1:2000"
+	defaultPollingInterval = 300
+)
+
+// Option is a function that sets config on the sampler.
+type Option func(options *config)
```

Comment:
```suggestion
// Option sets configuration on the sampler.
type Option interface {
	apply(*config)
}
```

See the [style guide for options](https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#option).

You can make an `optionFunc` implementation to deal with simple options:

```go
type optionFunc func(*config)

func (f optionFunc) apply(cfg *config) {
	f(cfg)
}
```